### DISCUSSION TITLE
Turkish users cannot register with Vercel.

### DISCUSSION THREAD

**Discussion by umutcandev** [vercel/community#1377] [Help] [answered]
### Summary

Hello,
I live in Turkiye(Turkey). To register with Vercel, i need to verify phone number. But Turkiye does not support telephone numbers. How can i solve this issue?

**Comment by swarnava** [comment, marked as answer]
Please email at registration@vercel.com using your registered email.

**Reply by umutcandev** [reply]
Thanks for the reply, I sent an email.

**Comment by hobik** [comment]
Hello I faced with same problem and sent mail to this address. Can you help me?

**Comment by mhmtbsrglu** [comment]
Re-open, also turkey have big community for vercel but we don't provide sms. We need sms verification for our country.
